### PR TITLE
doc: gsg: Updating SDK installation instructions to using `west sdk`

### PR DESCRIPTION
**Pull request by soburi**
SDK installation has been simplified by introducing the `west sdk`
command. I updated the documentation to use this.

### CODE REVIEW

**Review by pdgendt — DISMISSED** [review]

File: doc/develop/getting_started/index.rst [hunk 1]
```diff
@@ -386,12 +386,61 @@ contains toolchains for each of Zephyr's supported architectures, which
 include a compiler, assembler, linker and other programs required to build
 Zephyr applications.
 
-It also contains additional host tools, such as custom QEMU and OpenOCD builds
+For Linux, it also contains additional host tools, such as custom QEMU and OpenOCD builds
 that are used to emulate, flash and debug Zephyr applications.
 
-.. include:: ../toolchains/zephyr_sdk.rst
```
Comment (cfriedt) [inline doc/develop/getting_started/index.rst:392]: I haven't looked at this file yet, but are there any remaining references to it?

**Review by cfriedt — COMMENTED** [review]
This is a very nice simplification 👍

File: doc/develop/getting_started/index.rst (hunk 1 above)
Comment (cfriedt) [reply doc/develop/getting_started/index.rst:392]: Just revisited. It still has a fair bit of useful info.

**Review by cfriedt — APPROVED** [review]

**Review by pdgendt — APPROVED** [review]

**Review by benediktibk — APPROVED** [review]

**Comment by pdgendt** [comment]
Ping @kartben

**Review by kartben — APPROVED** [review]
Thank you very much, @soburi san!

**Pull request merged**